### PR TITLE
AB#3931 -- Updated FR text for type of application page

### PR DESCRIPTION
**Pull request by AdamP-HRSDC-RHDCC**
### Description
Updated FR text for type of application page

No changes to EN text

### Related Azure Boards Work Items
[AB#3931](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3931)

### Screenshots (if applicable)
image

### CODE REVIEW

File: frontend/public/locales/fr/apply.json [hunk 1]
```diff
@@ -90,25 +90,25 @@
...
-    "senior": "(FR) 65 or older",
-    "valid-disability-tax-credit": "(FR) 18 to 64 and have a valid Disability Tax Credit certificate",
-    "live-independently": "(FR) 16 or 17 and live independently from your parents or have a child who you are also applying for",
-    "apply-child": "(FR) Applying on behalf of your child",
-    "apply-child-eligibility": "(FR) You can apply on behalf of your child if:",
-    "sixteen-or-older": "(FR) You are 16 or older and the parent or legal guardian of the child; and",
-    "under-eighteen": "(FR) The child is under 18 years old",
+    "page-description": "La présentation des demandes au titre du Régime canadien de soins dentaires se fait par étapes.",
+    "apply-self": "Présenter une demande pour vous-même",
+    "apply-self-eligibility": "Vous pouvez actuellement présenter une demande pour vous-même si vous êtes:",
```
Comment (nicholas-ly) [inline frontend/public/locales/fr/apply.json:95]:
```suggestion
    "apply-self-eligibility": "Vous pouvez actuellement présenter une demande pour vous-même si vous êtes\u00a0:",
```

File: frontend/public/locales/fr/apply.json [hunk 2]
```diff
@@ -90,25 +90,25 @@
...
+    "senior": "65 ans ou plus;",
+    "valid-disability-tax-credit": "18 à 64 ans et avez un certificat valide de crédit d'impôt pour personnes handicapées;",
+    "live-independently": "16 ou 17 ans et vivez indépendamment de vos parents ou avez un enfant pour lequel vous faites également une demande.",
+    "apply-child": "Présenter une demande pour votre enfant",
+    "apply-child-eligibility": "Vous pouvez présenter une demande au nom de votre enfant si les deux conditions suivantes sont remplies:",
```
Comment (nicholas-ly) [inline frontend/public/locales/fr/apply.json:100]:
```suggestion
    "apply-child-eligibility": "Vous pouvez présenter une demande au nom de votre enfant si les deux conditions suivantes sont remplies\u00a0:",
```

File: frontend/public/locales/fr/apply.json [hunk 3]
```diff
@@ -90,25 +90,25 @@
...
     "form-instructions": "Qui présente la demande?",
-    "split-custody-summary": "(FR) If you split custody of a child",
-    "split-custody-detail": "(FR) If you have shared custody agreement for the child, eligibility for the plan and the amount covered is based on the parent or guardian that applies. The child's eligibility and amount covered may change if another parent or guardian applies for that child.",
-    "multiple-application": "(FR) Do not submit multiple applications for a single child. Only one parent should apply per child.",
+    "split-custody-summary": "Si vous partagez la garde d'un enfant",
+    "split-custody-detail": "Si vous avez une entente de garde partagée pour l'enfant, l'admissibilité au régime et le montant de la couverture sont déterminés par le parent ou le tuteur légal qui présente la demande. L'admissibilité de l'enfant et le montant couvert peuvent changer si un autre parent ou tuteur légal présente une demande pour cet enfant.",
+    "multiple-application": "Ne soumettez pas plusieurs demandes pour un même enfant : un seul parent devrait présenter la demande, et ce, pour chaque enfant.",
```
Comment (nicholas-ly) [inline frontend/public/locales/fr/apply.json:106]:
```suggestion
    "multiple-application": "Ne soumettez pas plusieurs demandes pour un même enfant\u00a0: un seul parent devrait présenter la demande, et ce, pour chaque enfant.",
```

File: frontend/public/locales/fr/apply.json [hunk 4]
```diff
@@ -90,25 +90,25 @@
...
     "radio-options": {
       "personal": "Je présente une demande pour <strong>moi</strong>",
```
Comment (nicholas-ly) [inline frontend/public/locales/fr/apply.json:108]:
```suggestion
      "personal": "Je présente une demande pour <strong>moi-même</strong>",
```

File: frontend/public/locales/fr/apply.json [hunk 5]
```diff
@@ -90,25 +90,25 @@
...
+    "apply-self": "Présenter une demande pour vous-même",
+    "apply-self-eligibility": "Vous pouvez actuellement présenter une demande pour vous-même si vous êtes:",
+    "senior": "65 ans ou plus;",
+    "valid-disability-tax-credit": "18 à 64 ans et avez un certificat valide de crédit d'impôt pour personnes handicapées;",
+    "live-independently": "16 ou 17 ans et vivez indépendamment de vos parents ou avez un enfant pour lequel vous faites également une demande.",
+    "apply-child": "Présenter une demande pour votre enfant",
+    "apply-child-eligibility": "Vous pouvez présenter une demande au nom de votre enfant si les deux conditions suivantes sont remplies:",
+    "sixteen-or-older": "vous avez 16 ans ou plus et êtes le parent ou le tuteur légal de cet enfant;",
+    "under-eighteen": "l'enfant est âgé de moins de 18 ans.",
     "form-instructions": "Qui présente la demande?",
```
Comment (nicholas-ly) [inline frontend/public/locales/fr/apply.json:103]:
```suggestion
    "form-instructions": "Pour qui présentez-vous la demande?",
```

Comment (AdamP-HRSDC-RHDCC) [reply frontend/public/locales/fr/apply.json]: Added missing translation

File: frontend/public/locales/fr/apply.json (hunk 4 above)
Comment (AdamP-HRSDC-RHDCC) [reply frontend/public/locales/fr/apply.json]: Added missing translation

File: frontend/public/locales/fr/apply.json (hunk 3 above)
Comment (AdamP-HRSDC-RHDCC) [reply frontend/public/locales/fr/apply.json]: Added spacing

File: frontend/public/locales/fr/apply.json (hunk 2 above)
Comment (AdamP-HRSDC-RHDCC) [reply frontend/public/locales/fr/apply.json]: Added spacing

File: frontend/public/locales/fr/apply.json (hunk 1 above)
Comment (AdamP-HRSDC-RHDCC) [reply frontend/public/locales/fr/apply.json]: Added spacing

**Review by nicholas-ly — APPROVED** [review]

**Pull request merged**